brd-android/app/src/main/java/com/breadwallet/ui/featureunavailable/FeatureUnavailableController.kt

```diff
@@ -232,6 +227,19 @@ class FeatureUnavailableController(
         }
     }
 
+    private fun veriffNeeded(profile: Profile?): Boolean {
```

Comment:
It is a good practice to name methods that return boolean with prefix verbs such as 'is' or 'has'.

```suggestion
    private fun isVeriffNeeded(profile: Profile?): Boolean {
```